CHANGES

```diff
@@ -82,6 +82,8 @@ Major release, unreleased
 - Fix incorrect JSON encoding of aware, non-UTC datetimes. (`#2374`_)
 - Template auto reloading will honor the ``run`` command's ``debug`` flag even
   if ``app.jinja_env`` was already accessed. (`#2373`_)
+- The ``flask`` command no longer requires that the ``FLASK_APP`` environment
+  variable have a ``.py`` extension. (`#2383`_)
```

Comment:
@bcongdon - you need to setup a target for this one:

`_#2383: https://github.com/pallets/flask/pull/2383`